java/src/main/java/ai/rapids/cudf/ColumnFilterOptions.java

```diff
@@ -29,18 +29,28 @@ public abstract class ColumnFilterOptions {
   // Names of the columns to be returned (other columns are skipped)
   // If empty all columns are returned.
   private final String[] includeColumnNames;
+  private final boolean[] readBinaryAsString;
 
   protected ColumnFilterOptions(Builder<?> builder) {
     includeColumnNames = builder.includeColumnNames.toArray(
         new String[builder.includeColumnNames.size()]);
+    readBinaryAsString = new boolean[builder.binaryAsStringColumns.size()];
+    for (int i = 0 ; i < builder.binaryAsStringColumns.size() ; i++) {
+      readBinaryAsString[i] = builder.binaryAsStringColumns.get(i);
+    }
   }
 
   String[] getIncludeColumnNames() {
     return includeColumnNames;
   }
 
+  boolean[] getConvertToBinaryRead() {
```

Comment:
The name of this method threw me a bit.  Switching between readBinaryAsString and getConvertToBinaryRead() is a little confusing.  Would be nice if we used a consistent name for what this is.